controllers/firewallrule_controller.go

```diff
@@ -403,12 +472,14 @@ func (r *FirewallRuleReconciler) clearFirewallRule(ctx context.Context, log logr
 	}
 
 	// Update status
+	rule.Status.LastTransitionTime = metav1.Now()
 	rule.Status = v1alpha1.FirewallRuleStatus{State: v1alpha1.FirewallRuleStateNone}
```

Comment:
That does not seem ok, you are setting LastTransitionTime and then overwriting it with an empty status.

Reply:
Well seen